ui/scripts/variables.js

```diff
@@ -0,0 +1 @@
+export const important_offers = ["kanopy", "hoopla", "criterionchannel", "prime", "amazon", "youtube", "netflix", "hbo", "itunes"];
```

Comment:
I think it's not "prime" (at least not in the latest justwatch list of possible values), but we can revisit later.